script.js

```diff
@@ -1,4 +1,218 @@
+
+// function to generate random number
+function random(max) {
+  let randomNum = Math.floor(Math.random() * max) + 1;
+  return randomNum;
+}
+// create function to generate random item
+function randomOption() {
```

Comment:
Same with this function- starting with verb makes it clearer that this is a function (where it is called)